beetmoverscript/templates/devedition_candidates_repacks.yml

```diff
@@ -19,6 +19,10 @@ mapping:
       s3_key: {{ platform }}/xpi/{{ locale }}.xpi
       destinations:
         - {{ platform }}/xpi/{{ locale }}.xpi
+    buildhub.json:
```

Comment:
We don't yet have the support for l10n in generating the buildhub.json so this will be useless for now. But I think we'll end up adding it later anyway once bug 1459302 is solved. So I think we can leave these 👍